service/pom.xml

```diff
@@ -22,6 +22,7 @@
         <module>local-storage-plugin</module>
         <module>confluent-schema-registry-plugin</module>
         <module>application</module>
+        <module>ibmmq-plugin</module>
```

Comment:
Please also update the README files to mention this plugin and examples similar to the existing ones.